Gemfile

```diff
@@ -9,11 +9,11 @@ ruby RUBY_VERSION
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem 'jekyll'
+gem 'jekyll', '~> 4.3', '>= 4.3.2'
```

Comment:
What's the reason for locking the version? Isn't it likely that we want to change again?